native/docs/manual-builds.md

```diff
@@ -29,24 +29,36 @@ found [here](build-configs.md).**
 - Install and setup XCode.
 - [Install and setup Fastlane](../../docs/cicd.md#fastlane-setup).
 - Read about [Apple Certificates](apple-certifcates.md).
+- To set up the certificates: have access to the `app-credentials` repo, and the Passbolt folder Integreat App
+  Publishing
 
 #### Certificates Setup
 
-- Prepare the necessary environment variables as
-  specified [here](../../docs/cicd.md#environment-variables-and-dependencies).
+- If none of your devices are logged in to the Apple account of the app team, warn someone with a logged in device
+  that you will need a code in a few minutes. That could be e.g. Steffi, Andy, or Leandra.
 
 - Install the certificates locally:
 
 ```bash
-cd ios && bundle exec fastlane certificates
+cd ios && fastlane match development (or whichever environment you need)
 ```
 
+- If you get an error that you can't clone the repo `app-credentials`, request access.
+
+- The password that you need to fill in twice is saved in Passbolt as Digitalfabrik Fastlane Match.
+
+- The Apple account that you need to sign in to is the one of the app team, the password is in Passbolt. This is also
```

Comment:
That is not correct, your personal account is linked to the organization. i never logged in with the app team account on my macbook

Reply:
Huh. I did. I don't think my personal Apple account is linked to the organization?

Reply:
Can you add some text about how to link a personal Apple account to the organization?

Reply:
You need to be either a) admin of the organization or b) be provided with the right to access "certificates, identifiers & profiles".
I added you as a admin. See the attached screenshot from appstoreconnect 
![leandra](https://github.com/user-attachments/assets/cd7129af-7a93-404a-a090-d73cd89385d5)

Reply:
Ah, cool, thanks!

Reply:
Oh, nice, without logging into the app-team account, we also don't need the second factor anymore 🥳